rfcs/0036-rfc-process-team-amendment.md

```diff
@@ -0,0 +1,202 @@
+      a summary comment trying to lay out the current state of the discussion
+      and major tradeoffs/points of disagreement.
+    * Before actually entering FCP, all members of the RFC Shepherd Team must
+      sign off the motion.
+10. The FCP lasts ten calendar days, so that it is open for at least 5 business
+days. It is also advertised widely, e.g. in NixOS Weekly and through Discourse
+announcements. This way all stakeholders have a chance to lodge any final
+objections before a decision is reached.
+11. In most cases, the FCP period is quiet, and the RFC is either merged or
+closed. However, sometimes substantial new arguments or ideas are raised, the
+FCP is canceled, and the RFC goes back into development mode.
+12. In case of acceptance, the RFC Steering Committee merges the PR into the
+`accepted`, in case of rejection into the `rejected` directory.
+
+
+![RFC Process](./0036-rfc-process.png)
+![Review Process](./0036-review-process.png)
+
+
+## The RFC life-cycle
+
+Once an RFC is accepted the authors may implement it and submit the feature as a
+pull request to the Nix or nixpkgs repo. Being accepted is not a rubber stamp,
+and in particular still does not mean the feature will ultimately be merged; it
+does mean that in principle all the major stakeholders have agreed to the
+feature and are amenable to merging it.
+
+Furthermore, the fact that a given RFC has been accepted implies nothing about
+what priority is assigned to its implementation, nor does it imply anything
+about whether a Nix/nixpkgs developer has been assigned the task of implementing
+the feature. While it is not necessary that the author of the RFC also write the
+implementation, it is by far the most effective way to see an RFC through to
+completion: authors should not expect that other project developers will take on
+responsibility for implementing their accepted feature.
+
+Minor modifications to accepted RFCs can be done in follow-up pull requests. We
+strive to write each RFC in a manner that it will reflect the final design of
+the feature; but the nature of the process means that we cannot expect every
+merged RFC to actually reflect what the end result will be after implementation.
+
+In general, once accepted, RFCs should not be substantially changed. Only very
+minor changes should be submitted as amendments. More substantial changes should
+be new RFCs, with a note added to the original RFC. Exactly what counts as a
+"very minor change" is up to the RFC Shepherd Team of the RFC to be amended, to
+be decided in cooperation with the RFC Steering Committee.
+
+
+## Members of the RFC Steering Committee
+
+In cooperation and discussion with Eelco Dolstra and all nominees the proposal
+for the first iteration of members of the RFC Steering Committee are:
+
+ - Eelco Dolstra (edolstra, niksnut)
+ - Shea Levy (shlevy)
+ - Domen Kožar (domenkozar)
+ - Jörg Thalheim (Mic92)
+ - Robin Gloster (globin)
+
```

Comment:
For convenience, it would be good to start mapping teams to github teams. Eg: @NixOS/rfc-steering . Not a blocker but I would like to see this happening.

Related, I should be removed write access on the rfcs repo since I haven't be selected in the initial team.

Reply:
Agreed about github teams (though naturally that should wait until after this RFC is accepted, if indeed it is).

Ambivalent about the commit bit question. So long as we trust everyone with the appropriate permissions to follow the relevant policies (including not accidentally running afoul of them), I don't think we need to be overly strict on access control.

Reply:
I agree. Also as @shlevy has said I don't necessarily thing that your commit bit should be removed. I don't see you trying to cause any harm etc.